sources/objectives/minimum_dissipation_objective_function.f90

```diff
@@ -255,22 +262,25 @@ end subroutine minimum_dissipation_objective_function_compute
 
   !> compute the sensitivity of the objective function with respect to $\chi$
```

Comment:
```suggestion
  !> compute the sensitivity of the objective function with respect to \f$\chi\f$
```
Doxygen does not directly understand `$` to mean a function. You have to tell it by using `\f$` instead.